apps/web/pages/event-types/index.tsx

```diff
@@ -258,9 +258,9 @@ export const EventTypeList = ({ group, readOnly, types }: EventTypeListProps): J
                           <Link href={"/event-types/" + type.id} passHref={true}>
```

Comment:
we should turn this into a component as its recurring

Reply:
@leog

Reply:
heh was just gonna mention about componenting it out! Great candidate for first ui component to reuse

Reply:
for the `Link`? GH diff shows me just that for this comment